### PR TITLE
deps: ember-ajax@0.7.1

### PR DESCRIPTION
**Pull request by acburdine**
Converts ic-ajax usage to ember-ajax usage

TODOs:
- [x] Initial switch
- [x] Fix api error handling through ember-ajax
- ~~Revisit modal ajax usage (to be completed once #6088 is merged)~~
- [x] Tests?

### CODE REVIEW

File: core/client/app/controllers/settings/labs.js [hunk 1]
```diff
@@ -68,6 +67,7 @@ export default Controller.extend({
                 // TODO: keep as notification, add link to view content
                 notifications.showNotification('Import successful.', {key: 'import.upload.success'});
             }).catch((response) => {
+                // TODO: revisit
```
Comment (kevinansfield) [inline core/client/app/controllers/settings/labs.js]: Does anything further need to be done here or is this working as-is?

**Comment by kevinansfield** [comment]
:+1: LGTM. Ready to merge once I hear back about the `TODO: revisit` query above. Thanks! :smile:

File: core/client/app/controllers/settings/labs.js (hunk 1 above)
Comment (acburdine) [inline core/client/app/controllers/settings/labs.js]: Oops 😛 Will get that fixed up momentarily.

**Comment by kevinansfield** [comment]
Sorry about adding something else to the todo list before merging but I think the readability could be improved by pulling the `ghostPaths` url generation out of the request invocation, eg. replacing:

```javascript
return this.get('ajax').request(this.get('ghostPaths.url').api('authentication', 'invitation'), { ... }).then(() => {
});
```

with:
```javascript
let invitationUrl = this.get('ghostPaths.url').api('authentication', 'invitation');
return this.get('ajax').request(invitationUrl, { ... }).then(() => {
});
```

**Pull request merged**